astropy/coordinates/baseframe.py

```diff
@@ -156,6 +156,69 @@ def __new__(mcls, name, bases, members):
                 'Could not find all expected BaseCoordinateFrame class '
                 'attributes.  Are you mis-using FrameMeta?')
 
+        # Unless overridden, velocity name defaults are:
+        #   * pm_{lon}_cos{lat} for SphericalCosLatDifferential
+        #   * radial_velocity for any d_distance, including RadialDifferential
+        #   * v_{x,y,z} for d_{x,y,z} for CartesianDifferential
+        # where {lon} and {lat} are the names of the angular components
+        if repr_info is None:
+            repr_info = dict()
+
+        if r.SphericalRepresentation in repr_info:
+            sph_mappings = repr_info[r.SphericalRepresentation]
+            component_map = dict([(m.reprname, m.framename)
+                                  for m in sph_mappings])
+
+        else:
+            component_map = dict()
```

Comment:
Can't this just be `component_map = {'lat': 'lat', 'lon': 'lon'}` (and then delete the `setdefault`).
and then delete the `setdefault` below? I.e., it is possible for `repr_info[r.SphericalRepresentation]` to not contain those (and then an error would seem right?)

Reply:
Yes, I think that's right